users/mverteuil/mverteuil.c

```diff
@@ -0,0 +1,42 @@
+#include QMK_KEYBOARD_H
+#include "mverteuil.h"
+
+void register_left_brace() {
+    register_code(KC_LSHIFT);
+    register_code(KC_LBRACKET);
```

Comment:
This can be replaced by: 
```c
register_code16(LSFT(KC_LBRACKET));
```
or
```c
register_code16(KC_LEFT_CURLY_BRACE);
```

https://github.com/qmk/qmk_firmware/blob/master/quantum/quantum_keycodes.h#L550-L612

Reply:
AHHHHH man I love you! I can just use that directly in `keymap.c` and skip the wrappers altogether ;)